awx/lib/awx_display_callback/module.py

```diff
@@ -308,8 +308,10 @@ def v2_playbook_on_stats(self, stats):
             changed=stats.changed,
             dark=stats.dark,
             failures=stats.failures,
+            ignored=stats.ignored,
```

Comment:
Does this work in older ansible versions, or do we need to do a `getattr(stats, 'ignored', 0)` here?

Reply:
We _do_ need to do something like this. Gave it a try and works for both pre-2.8 and 2.8.